### PR TITLE
Build: Swap order of e2e tests around

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: e2e pnp tests are failing

## What I did

- swap order of e2e tests around

this makes them pass, we think that perhaps some tests are leaking into each other.
We plan to migrate off of cypress in the future,
so I'm not going to spend more time on improving this.

### CODE REVIEW

File: addons/docs/package.json [hunk 1]
```diff
@@ -77,6 +77,7 @@
     "fast-deep-equal": "^3.1.3",
     "global": "^4.4.0",
     "lodash": "^4.17.21",
+    "regenerator-runtime": "^0.13.7",
```
Comment (yannbf) [inline addons/docs/package.json:80]: why is this necessary in addon-docs?

**Comment by yannbf** [comment]
Nice stuff @ndelangen <3

**Review by shilman — APPROVED** [review]
awesome 👏

**Comment by ndelangen** [comment]
Just for future reference, turns out that there's a something not quite right here: 
https://github.com/storybookjs/storybook/blob/450fb4aa30f6cdb5d4ba92c8ebbd31d4f328997e/cypress/support/commands.js#L101-L110

There seems to be some order of operation error, and that's the true cause some tests aren't passing.. the wrong story is selected, and thus (as one would expect) the test fails.

By waiting a bit longer, the story is correctly selected, and the test proceeds as designed...
